packages/cf-style-provider/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
please lock dependencies with yarn.lock. We don't want failed CI builds every time fela upgrades